src/frontend/apps/desk/src/components/BoxButton.tsx

```diff
@@ -0,0 +1,28 @@
+import { ComponentPropsWithRef, forwardRef } from 'react';
+
+import { Box, BoxType } from './Box';
+
+export type BoxButtonType = ComponentPropsWithRef<typeof BoxButton>;
+
+const BoxButton = forwardRef<HTMLDivElement, BoxType>(
```

Comment:
luv this component, it will be useful !! 
Might be great to add a bit of docstring?